### PR TITLE
Feat: Implement Max.Redirects & Accepted Status Codes

### PR DESCRIPTION
**Pull request by chakflying**
This PR implements options in each monitor for:
- The maximum number of redirects to follow,
- The ranges of status codes to accept.

image

image

I'm a bit torn with the variable naming, since I want to follow the existing naming conventions, but true camel case should be `maxRedirects`.

`multiselect` support for Vue 3 has not hit stable yet, but functionality looks fine to me.

Also, sorry for bundling a bit of styling change, but I got really triggered by the lack of spacing 😂

### CODE REVIEW

**Comment by louislam** [comment]
Appreciate with the work!

The only problem is I think in this way, it cannot cover @Hyurt's use case. https://github.com/louislam/uptime-kuma/issues/11#issuecomment-890578686

**Comment by chakflying** [comment]
Currently the user can select a mix of preset ranges and individual codes, so `"200-299", "401", "402", "403"` would work.

I guess the alternative would be to give user a text box and let them enter arbitrary ranges, but I'm not sure if having to handle parsing or client-side validation is worth the effort.

**Comment by louislam** [comment]
> Currently the user can select a mix of preset ranges and individual codes, so `"200-299", "401", "402", "403"` would work.
> 
> I guess the alternative would be to give user a text box and let them enter arbitrary ranges, but I'm not sure if having to handle parsing or client-side validation is worth the effort.

Nice, I do not checkout yet, I thought it accepts range only.🤣

File: src/pages/EditMonitor.vue [hunk 1]
```diff
@@ -124,18 +142,31 @@
 <script>
 import NotificationDialog from "../components/NotificationDialog.vue";
 import { useToast } from "vue-toastification"
+import VueMultiselect from "vue-multiselect"
 const toast = useToast()
 
 export default {
     components: {
         NotificationDialog,
+        VueMultiselect
```
Comment (Saibamen) [inline src/pages/EditMonitor.vue:151]:
```suggestion
        VueMultiselect,
```

**Comment by louislam** [comment]
Updated the color to match the theme.

Btw, I updated sqlite3, you may need to run "npm install" again.

**Pull request merged**